mysite/ct/admin.py

```diff
@@ -1,19 +1,161 @@
 from django.contrib import admin
+from django.forms.models import ModelForm
+
 import ct.models
 
 
+class BaseForm(ModelForm):
+    """
+    Base class for admin forms.
+    """
+    def __init__(self, *args, **kwargs):
+        super(BaseForm, self).__init__(*args, **kwargs)
+
+        for field_name in ('addedBy', 'author'):
+            if field_name in self.fields and not self.initial.get(field_name):
+                self.initial[field_name] = self.current_user
+
+
+class ConceptForm(BaseForm):
+    """
+    Form for AdminConcept.
+    """
+    class Meta:
+        models = ct.models.Concept
+
+
+class ConceptGraphForm(BaseForm):
+    """
+    Form for AdminConceptGraph.
+    """
+    class Meta:
+        models = ct.models.ConceptGraph
+
+
+class LessonForm(BaseForm):
+    """
+    Form for AdminLesson.
+    """
+    class Meta:
+        models = ct.models.Lesson
+
+
+class ConceptLinkForm(BaseForm):
+    """
+    Form for AdminConceptLink.
+    """
+    class Meta:
+        models = ct.models.ConceptLink
+
+
+class UnitLessonForm(BaseForm):
+    """
+    Form for AdminUnitLesson.
+    """
+    class Meta:
+        models = ct.models.UnitLesson
+
+
+class UnitForm(BaseForm):
+    """
+    Form for AdminUnit.
+    """
+    class Meta:
+        models = ct.models.Unit
+
+
+class ResponseForm(BaseForm):
+    """
+    Form for AdminResponse.
+    """
+    class Meta:
+        models = ct.models.Response
+
+
+class StudentErrorForm(BaseForm):
+    """
+    Form for AdminStudentError.
+    """
+    class Meta:
+        models = ct.models.StudentError
+
+
+class CourseForm(BaseForm):
+    """
+    Form for AdminCourse.
+    """
+    class Meta:
+        models = ct.models.Course
+
+
+class CourseUnitForm(BaseForm):
```

Comment:
After some search, we've decide that we can remove forms declarations completely and pass BaseForm while register AdminModel. This is possible, because django will put it in modelform_factory in any cases:
https://github.com/django/django/blob/master/django/contrib/admin/options.py#L637

Also, admin.register can take multiple models 

So, final code could looks like this:

```python
models = [
    ct.models.Concept, ct.models.ConceptGraph, ct.models.Lesson, 
    ct.models.ConceptLink, ct.models.UnitLesson, ct.models.Unit,
    ct.models.Response, ct.models.StudentError, ct.models.Course,
    ct.models.CourseUnit
]

@admin.register(*models)
class AdminModel(BaseAdmin):
    form = BaseForm
```